playkit/src/main/java/com/kaltura/playkit/plugins/KalturaStatsPlugin.java

```diff
@@ -179,6 +183,12 @@ protected void onUpdateMedia(PKMediaConfig mediaConfig) {
         } else {
             baseUrl = DEFAULT_BASE_URL;
         }
+        if (Utils.isJsonObjectValueValid(pluginConfig, "timerInterval")) {
+            timerInterval = pluginConfig.getAsJsonPrimitive("timerInterval").getAsInt() * (int)Consts.MILLISECONDS_MULTIPLIER;
```

Comment:
In that new Config class, the `timerInterval` field will be called `timerIntervalMillis`. Let the application translate.